packages/amplify_api_rest_plugin_interface/README.md

```diff
@@ -0,0 +1,15 @@
+# amplify_api_rest_plugin_interface
+
+A new flutter plugin project.
```

Comment:
Please update.

Reply:
Thanks I'll make a note for Ashish and I to follow up on this